src/cpp/src/text2image/schedulers/ddim.cpp

```diff
@@ -0,0 +1,208 @@
+// Copyright (C) 2023-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cassert>
+#include <random>
+#include <fstream>
+#include <iterator>
+
+#include "text2image/schedulers/ddim.hpp"
+
+
+namespace {
+
+// https://gist.github.com/lorenzoriano/5414671
+template <typename T, typename U>
+std::vector<T> linspace(U start, U end, size_t num, bool endpoint = false) {
```

Comment:
It seems duplication of: https://github.com/openvinotoolkit/openvino.genai/blob/e9d4caee4338de1cca678fb9406f011fd1f01e9c/src/cpp/src/text2image/schedulers/lcm.cpp#L15
Can it be reused?